circle.yml

```diff
@@ -8,7 +8,6 @@ checkout:
 
 dependencies:
   post:
-    - bundle exec rake lint:install
```

Comment:
Are the invocations of `bundle exec rake lint:*` below still necessary too, if they now happen to be fully integrated as a Build Phase in the Xcode project, and thus will already be run by `bundle exec rake xcode:test` (I think)?

```
 test:
   override:
     - bundle exec rake lint:code
     - bundle exec rake lint:tests
     - bundle exec rake lint:output
```

Reply:
The xcode build phases are wrapped in a
```bash
if [[ ! $CI && ! $NO_CODE_LINT ]]; then
```

The CI steps are separate so that they are separate "steps", and can happen in parallel, have separate logs, etc...

Reply:
Right. We should just be careful to be sure to maintain both (Build Phases vs Rake tasks) in sync. Alternatively, we could make those tasks simple bash scripts in a `scripts/` folder, then invoking those bash scripts both in the Build Phases (wrapped in `if !$CI`) and in `circle.yml`

Reply:
Maybe keep the rake tasks as wrappers for the bash scripts? That way we can keep having a unified place for triggering "stuff" from CI and CLI (locally).

Reply:
Totally!